.github/workflows/validate-integration-files.yml

```diff
@@ -0,0 +1,71 @@
+name: Validate integration files
+
+on:
+  pull_request:
+    paths:
+      - 'integrations/**'
+
+jobs:
+  validate-files:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Setup
+      run: |
+        pip install toml-cli yq
+
+    - name: Check Ocean version 🌊
+      run: |
+        changed_dirs=$(git diff --name-only origin/main origin/${{ github.head_ref }} | grep 'integrations/' | cut -d'/' -f 1,2 | sort -u)
+        package_version=$(curl -s https://pypi.org/pypi/port-ocean/json | jq -r '.info.version')
+        for dir in $changed_dirs; do
+          pyproject_file=$(find $dir -name 'pyproject.toml' -not -path "**/.venv/*")
+          if [ -n "$pyproject_file" ]; then
```

Comment:
Probably better to check the exit code of the find command (it will return 0 if something was found, and 1 if there were no results)

Reply:
i already save the result to a var so i dont think it really matters